added some bold text, full stop and some other stuff. (#2246)
EddieHubCommunity/hacktoberfest-practice: README.md

<!-- markdownlint-disable-next-line -->
> **Note** **This repository is _not_ included in the Hacktoberfest event, as it is for practice only!**

> We have other  [repositories](https://github.com/orgs/EddieHubCommunity/repositories) in the organization that you can contribute to. If you would like to join our GitHub organisation, raise an [issue]( https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) on this repo EddieHubCommunity-Support and you can also join the EddieHub  [Discord](http://discord.eddiehub.org/) channel.

Reviewer: There are a few extra spaces here that could be removed.
```suggestion
We have other [repositories](https://github.com/orgs/EddieHubCommunity/repositories) in the organization that you can contribute to. If you would like to join our GitHub organisation, raise an [issue](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) on the support repository, and you can also join the EddieHub's [Discord](http://discord.eddiehub.org/) community.
```

    commit -->|Finally|creatpr((Create a Pull Request))
    
 ```

> **1. Fork the project :**

Reviewer: I think it looks better without space before `:` (same for all the other instances).
```suggestion
**1. Fork the project:**
```

- Click the gray <kbd>Fork</kbd> button at the top right of this page. This creates your copy of the project and saves it as a new repository in your GitHub account.

> **2. Create a New Branch :**

Reviewer: ```suggestion
**2. Create a New Branch:**
```

- On your new repository's page, click the gray main button in the upper left to reveal a dropdown menu.
- Enter the name of your new branch in the text box. (Branch names usually refer to what is being changed. Example: nameAdd).
-Click on Create branch <new branch name>, which will automatically take you to your new branch. You can make edits on the main branch, but this may cause issues down the line. The best practice is to create a new branch for each separate issue you work on. That way your main branch remains in sync with Eddie's main branch.

> **3. Edit :**

Reviewer: ```suggestion
**3. Edit:**
```

- On the top right of the Readme file, click on the pencil icon to edit the file by adding your name.
- After editing the Readme file, add a commit message and click on the green button saying "Commit Changes". Make sure you have selected the branch you have created.

> **4. Raise a Pull Request :**

Reviewer: ```suggestion
**4. Raise a Pull Request:**
```

  - [Ayushi Vamne](https://github.com/Ayushi15-Vamne) 
  - [Aziz Prabowo](https://github.com/azizp128)
  

> | [`Back To Top`](#contents) |

Reviewer: I don't think code backticks are needed for this (same for all the other instances). Not sure what others think, though.

Author: Ok..and thanks for replacement of those mistakes.

  - [Ayushi Vamne](https://github.com/Ayushi15-Vamne) 
  - [Aziz Prabowo](https://github.com/azizp128)
  

> | [`Back To Top`](#contents) |

Reviewer: Hmm, I think we should remove the code backticks, as well as in all the other instances of this.

  - [Bunde Uji](https://github.com/bunde-uji)
  - [Burhanuddin Raja](https://github.com/BurhanRaja)

> | [`Back To Top`](#contents) |

Reviewer: I suggest not using code backticks here (same in other parts of the file).
```suggestion
| [Back To Top](#contents) |
```
